CONTRIBUTING.md

```diff
@@ -130,6 +130,24 @@ cypress window will open locally - select test file to run the tests
 
 cypress window will open locally - select test file to run the tests
 
+#### Test UX and SDK changes
+
+For any new change, which involves changing any of the python SDK components and UI components, the manual testing of the code change can be done using the following steps:
+
+1. git clone <https://github.com/microsoft/responsible-ai-toolbox>
+2. `cd responsible-ai-toolbox` (It is recommended to create a new virtual environment and install the dependencies)
+3. You should commit all your current set of changes for SDK and UX using `git commit`.
+4. Clean all untracked files using `git clean -fdx`
```

Comment:
Did you want to add that this may not be required in all cases? It's definitely a huge time saver... I'll leave it up to you 🙂

Reply:
I don't know if 'git clean' applies to all the testing scenario.